src/services/infra/InfraService.ts

```diff
@@ -1,15 +1,39 @@
 import { SubDomainSettings } from "aws-sdk/clients/amplify"
 import Joi from "joi"
-import { Err, err, errAsync, Ok, ok, okAsync, Result } from "neverthrow"
+import {
+  Err,
+  err,
+  errAsync,
+  fromPromise,
+  Ok,
+  ok,
+  okAsync,
+  Result,
+  ResultAsync,
+} from "neverthrow"
 
 import { config } from "@config/config"
 
 import { Site } from "@database/models"
 import { User } from "@database/models/User"
 import { SiteLaunchMessage } from "@root/../microservices/site-launch/shared/types"
-import { SiteStatus, JobStatus, RedirectionTypes } from "@root/constants"
+import UserWithSiteSessionData from "@root/classes/UserWithSiteSessionData"
+import {
+  SiteStatus,
+  JobStatus,
+  RedirectionTypes,
+  REDIRECTION_SERVER_IP,
+} from "@root/constants"
+import MissingSiteError from "@root/errors/MissingSiteError"
+import MissingUserEmailError from "@root/errors/MissingUserEmailError"
+import SiteLaunchError from "@root/errors/SiteLaunchError"
 import logger from "@root/logger/logger"
 import { AmplifyError } from "@root/types/amplify"
+import {
+  DnsResultsForSite as SiteDnsResults,
```

Comment:
could we do a rename on the type?

Reply:
opps sorry forgot to refactor this, done here 198e2a257522b08f6296a7c4bd21a135e67ed325